### ISSUE TITLE
Not compatible with v1.70.3 DWIN Display Firmware

### ISSUE THREAD

**Issue by onurtirpan** [KerseyFabrications/marlin_e5p#8]
The newer display firmware doesn't support your firmware mod, if you could share your compatible display dwin firmware or update the firmware to be compatible with 1.70.3, new people with newer firmware could also benefit your changes.

**Comment by syfyghter** [comment]
You should look here [https://drive.google.com/drive/folders/16596EvFWv9C9ClbavmHPc4J3MJ4_N3h_](url)

**Comment by onurtirpan** [comment]
> https://drive.google.com/drive/folders/16596EvFWv9C9ClbavmHPc4J3MJ4_N3h_

I am not sure if you understood my problem. 

Kersey shared his firmware for the board, which is what I want. 
But, the DWIN firmware(s) for the display is not compatible with the Kersey framework. At least, none of the display firmware worked when I tried them.

**Comment by syfyghter** [comment]
Kersey made no changes to the code for the display, as far as i know. The combination of Kersey's Firmware and the dwin software that comes with the opensource files from Creality (1.70.1 BL) for the screen works for me! This is the 2 language software (chinese and english).

**Comment by MikeUSNRet** [comment]
I just purchased an E5+ the end of December.  The Creality Firmware 1.70.3 BL is a soup sandwich and full of bugs.  Downloaded the firmware here after a recommendation and it won't even load whether using Cura or Xloader.  Have read on Reddit of others having this same issue and that apparently it won't work with the newer v2.2 mainboards being shipped out now due to a change in chipsets.

**Comment by syfyghter** [comment]
Hi,
can you discribe the error while uploading the firmware? Do you have the link to reddit? What's the problem with the stock firmware for example?

**Comment by MikeUSNRet** [comment]
no.  don't have that link available and do not have the time to wade through a backlog of posts over the past month sadly.   As for uploading the firmware.  In cura it will never read the hex file to begin with to begin an upload of it.   Selected the .hex,  clicked on update firmware and waited and waited and waited, made a sandwich and was still waiting, washed the dishes and still waiting..... lol    Tried to use Xloader and it ran the update, but said Upload Failed  repeatedly.    As for the stock firmware 1.70.3 BL ,   babysetting the Z while printing can only be changed in 0.1mm increments which is too much when previous versions allowed 0.01mm in babysteps.  Creality disabled the Autolevel slider (even though most turn it off anyway).  People experiencing anywhere from 5 - 22mm variance from corner to corner when runnign autolevel for a bed mesh.  NOBODY has a bed that far out unless the thing has been bent in half almost.  Creality thinks this error is a defective BL or CR Touch.  I ended up installing Insanity Automation Firmware and screenset.  No CR Touch issues.  Baby steps in 0.02mm increments, Autoleveling available and wow, my overall variance is just 0.04mm vice the 18mm I personally was getting with the stock FW.

**Comment by jbeima** [comment]
I was wondering on the .1 as well, that is so high for baby stepping and is not working for me... I need it down much lower like 0.02...

I am seeing about a .6 from one side to another, which I don't believe is the case, from experience with other printers, and what I see while I work with this printer.

I can also confirm that my printer came with 1.70.3BL, but it has a CR Touch not a BL Touch on it.
